test/controllers/TimeoutControllerSpec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import play.api.http.Status
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import testsupport.{ItSpec, WireMockSupport}
+
+import java.net.URLEncoder.encode
+
+class TimeoutControllerSpec extends ItSpec {
+  val controller: TimeoutController = app.injector.instanceOf[TimeoutController]
+
+  "TimeoutController" - {
+    ".killSession" - {
+      "should display 'delete-answers' page with sign in again url to TimeoutController.signInAgain" in {
+        val result = controller.killSession(FakeRequest())
```

Comment:
if you want to test the session is actually reset as well you could pass in a session with dummy values with something like this:
```scala
FakeRequest().withSession("key" -> "value")
```
and then do an assertion like
```scala
session(result) shouldBe Session()
```
It might not be exactly as above since I'm just blindly coding here, but it should be quite similar

Reply:
added kill session test as suggested